python/foglamp/services/core/routes.py

```diff
@@ -119,6 +120,9 @@ def setup(app):
     # Get Syslog
     app.router.add_route('GET', '/foglamp/syslog', support.get_syslog_entries)
 
+    # Get Plugin
+    app.router.add_route('GET', '/foglamp/plugins/installed', plugin_discovery.get_plugins_installed)
```

Comment:
Tests please. and what about available for install list

Reply:
As @MarkRiddoch mentioned, it will taken up in a future sprint.